tensorflow_serving/tools/docker/tests/BUILD

```diff
@@ -46,6 +46,38 @@ sh_test(
     deps = [":docker_test_lib"],
 )
 
+sh_test(
+    name = "unittest_dockerfile_mkl",
+    size = "medium",
```

Comment:
Does this take a while to start up (GPU does). Otherwise this could be "small"

Reply:
it was faster, it didn't take much time while testing.